packages/traceability-schemas/schemas/Observation.json

```diff
@@ -30,6 +30,12 @@
       "title": "Measured Value",
       "description": "The measurement of an Observation.",
       "$ref": "https://w3id.org/traceability/schemas/MeasuredValue.json"
+    },
+    "date": {
+      "$comment": "{\"term\": \"date\", \"@id\": \"https://schema.org/Date\"}",
```

Comment:
This should probably be DateTime similar to existing schema calls, for example: 
[AgActivity](https://github.com/w3c-ccg/traceability-vocab/blob/fd05e51d64585908d2ef382a1304b449f757948c/packages/traceability-schemas/schemas/AgActivity.json#L44)
[IntentToSell](https://github.com/w3c-ccg/traceability-vocab/blob/fd05e51d64585908d2ef382a1304b449f757948c/packages/traceability-schemas/schemas/IntentToSell.json#L41)

Reply:
agreed

Reply:
For sure, @mkhraisha. +1